packages/code-block/partials/code-lines/style.module.css

```diff
@@ -12,6 +12,13 @@ pre.pre {
   padding: 0;
   margin: 0;
   background-color: transparent;
+
+  /* Prevent shell-symbols from being selected, for better select-copy-paste */
+  &:global(.language-shell-session) {
+    & :global(.token.shell-symbol) {
+      user-select: none;
```

Comment:
CSS fix for the "select-copy-paste" use case. This applies to all code-blocks with `$` characters (when correctly identified as `.shell-symbol`, which of course requires highlighting).